apps/src/maze/api.js

```diff
@@ -312,8 +312,8 @@ exports.getPumpkin = API_FUNCTION(function (id) {
   Maze.subtype.getPumpkin(id);
 });
 
-exports.getBean = API_FUNCTION(function (id) {
-  Maze.subtype.getBean(id);
+exports.getLettuce = API_FUNCTION(function (id) {
+  Maze.subtype.getLettuce(id);
```

Comment:
I'd say the primary consideration for the API is what it makes the user's code look like; although it ends up being more work for us this way, the resulting code that the kids see is much nicer.